app/graphql/types/block.rb

```diff
@@ -0,0 +1,20 @@
+class Types::Block < Types::BaseObject
+  implements Types::Interface::WithTimestamps
+
+  description 'A blocked user entry of an Account.'
+
+  field :id, ID, null: false
+
+  field :user, Types::Profile,
+    null: false,
+    description: 'User who blocked.',
+    method: :user
+
+  field :blocked_user, Types::Profile,
+    null: false,
+    description: 'User who got blocked.'
+
+  def blocked_user
+    Loaders::RecordLoader.for(User).load(object.blocked_id)
+  end
```

Comment:
You probably also want this for `user` (not that it's likely to ever be navigated the other direction, really just in case)